bundles/org.openhab.binding.clementineremote/src/main/resources/OH-INF/binding/binding.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="clementineremote" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:binding="https://openhab.org/schemas/binding/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/binding/v1.0.0 https://openhab.org/schemas/binding-1.0.0.xsd">
+
+	<name>Clementine Remote Control</name>
+	<description>This is a binding to remotely control Clementine Player instances.</description>
+
+</binding:binding>
```

Comment:
Please convert to the new `addon.xml`.

Reply:
This `addon.xml` is not yet common knowledge and the PR to adapt the developer docs is not merged yet, so if you need more details, the corrosponding issue is here: https://github.com/openhab/openhab-core/issues/2058
But you can also update your branch and look at another binding's `addon.xml` and adapt it to this initial contribution.

Reply:
Details are documented here: https://next.openhab.org/docs/developer/addons/addon.html#add-on-definitions